Looms.PoS/Looms.PoS.Application/Features/Product/Commands/CreateProduct/CreateProductCommand.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Looms.PoS.Application.Abstracts;
+using MediatR;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Looms.PoS.Application.Features.Product.Commands.CreateProduct;
+
+public record CreateProductCommand : LoomsHttpRequest, IRequest<IActionResult>
+{
+    public CreateProductCommand(HttpRequest request) : base(request)
+    {
+        //TODO: idk where to check if this has a body, but from what i understand it doesnt 
```

Comment:
This doesn't need a body

Reply:
fixed in "fixed indentation/layout problems, changed Variation to Variation request and made create and update requests take IEnumerable<VariationRequest>"